app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -41,6 +42,7 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener {
         super.onCreate(savedInstanceState)
 
         initAmazonFactory()
+        CustomTilesAccessor.initWithCache(CustomTilesAccessor.getCustomTilesCache(this))
```

Comment:
If we ever recreate MainActivity, this will be called again and it will break. Either:
1) Make sure init is only called once
2) Call this from FocusApplication.onCreate (seems like a bad idea from a perf perspective).

Also, calling this here will break if we ever refactor the code to enter from another Activity – if you made the cache lazily-initted in each of the public accessor methods, it'd avoid this problem (and the class would have fewer dependencies). If you decide to leave it as is, please document that this method needs to be called in the class javadoc.

Reply:
I don't think this will break, because it just refreshes the cache.

But I'll switch this to getInstance anyway - probably good to try it out and if it doesn't work out I'll remember :P